server/utils/applications/utils.ts

```diff
@@ -199,13 +194,19 @@ const isInapplicable = (application: Application): boolean => {
     return true
   }
 
-  if (shouldPersonBePlacedInMaleAp === 'no') {
-    return true
-  }
-
   return false
 }
 
+const isWomensAp = (application: Application): boolean => {
```

Comment:
As per previous comment, `ap` could be ambiguous here -- let's be explicit and call this `isWomensApplication`